### PR TITLE
Updated translation

### PR DESCRIPTION
**Pull request by Blask0**
Fixed some translations

### CODE REVIEW

File: lib/l10n/intl_es_ES.arb [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+  "light": "Claro",
   "english": "Inglés",
   "spanish": "Español",
   "french": "Francés",
   "russian": "Ruso",
   "simplifiedChinese": "Chino (Simplificado)",
   "portuguese": "Portugués",
   "bow": "Arco",
-  "claymore": "Mandoble",
+  "claymore": "Mina",
```
Comment (Wolfteam) [inline lib/l10n/intl_es_ES.arb:12]: Aca deberia seguir siendo "Mandoble"

File: lib/l10n/intl_es_ES.arb [hunk 2]
```diff
@@ -134,19 +134,19 @@
   "brandNew": "Nuevo",
   "noInternetConnection": "No hay conexión hay internet",
   "wishSimulator": "Simulador de Deseos",
-  "tryYourLuck": "Prueba tu suerte con este simulador de deseos",
-  "bonus": "Bonos",
+  "tryYourLuck": "Prueba suerte con este simulador de deseos",
+  "bonus": "Bonuss",
```
Comment (Wolfteam) [inline lib/l10n/intl_es_ES.arb:138]: Typo

**Review by Wolfteam — COMMENTED** [review]
Just a few comments

File: lib/l10n/intl_es_ES.arb (hunk 2 above)
Comment (Blask0) [reply lib/l10n/intl_es_ES.arb:138]: My bad

File: lib/l10n/intl_es_ES.arb (hunk 1 above)
Comment (Blask0) [reply lib/l10n/intl_es_ES.arb:12]: Ok!

**Comment by Wolfteam** [comment]
Please don't forget to update this pr

**Pull request merged**